Packs/MISP/Integrations/MISPV3/MISPV3.yml

```diff
@@ -2601,6 +2601,112 @@ script:
     - contextPath: MISP.Attribute.Sighting.Type
       description: Sighting type.
       type: String
+  - arguments:
+    - default: false
+      description: UUID of the attribute to be updated.
+      isArray: false
+      name: attribute_uuid
+      required: true
+      secret: false
+    - default: false
+      description: 'Attribute type. For example: "md5", "sha1", "email", "url".'
+      isArray: false
+      name: type
+      required: false
+      secret: false
+    - default: false
+      description: 'Attribute category. For example: "Other", "Person", "Attribution", "Payload type".'
```

Comment:
Same question

Reply:
Unfortunately no, it can be very long list of MISP categories (and if I am not wrong a custom category can be added via the MISP UI).